Packages/vcs/Lib/vcsvtk/boxfillpipeline.py

```diff
@@ -0,0 +1,383 @@
+from .pipeline2d import Pipeline2D
+from .. import vcs2vtk
+
+import numpy
+import vcs
+import vtk
+import warnings
+
+class BoxfillPipeline(Pipeline2D):
+    """Implementation of the Pipeline interface for VCS boxfill plots.
+
+    Internal variables:
+        - self._legend: Contour labels.
```

Comment:
Ah, this is the variable named `legend` in the original `plot2D`, so I just kept the name. I'll rename it.